app/assets/stylesheets/govuk_publishing_components/components/govspeak/content_blocks/_contact.scss

```diff
@@ -0,0 +1,14 @@
+@import "govuk/core/typography";
+
+.govspeak, // Legacy class name that's still used in some content items - needs to be kept until `.govspeak` is removed from the content items.
```

Comment:
It might be that you don't need to nest your new classes inside the `.govspeak` class - this is a (possibly overused) convention to prevent buttons that are links from being styled incorrectly.

Reply:
That seems to work! 👍